docs/source/user-guide/concepts/pkg-specs.rst

```diff
@@ -0,0 +1,394 @@
+===========================
+Conda package specification
+===========================
+
+.. contents::
+   :local:
+   :depth: 1
+
+
+.. _package_metadata:
+
+Package metadata
+================
+
+The ``info/`` directory contains all metadata about a package.
+Files in this location are not installed under the install
+prefix. Although you are free to add any file to this directory,
+conda only inspects the content of the files discussed below.
+
+Info
+----
+
+* files
+
+  * a list of all the files in the package (not included in ``info/``)
+
+* ``index.json``
+
+  * metadata about the package including platform, version,
+    dependencies, and build info
+  
+.. code-block:: bash
+
+  {
+    "arch": "x86_64",
+    "build": "py37hfa4b5c9_1",
+    "build_number": 1,
+    "depends": [
+      "depend > 1.1.1"
+    ],
+    "license": "BSD 3-Clause",
+    "name": "fun-packge",
+    "platform": "linux",
+    "subdir": "linux-64",
+    "timestamp": 1535416612069,
+    "version": "0.0.0"
+  }
+
+* ``paths.json``
+
+  * a list of files in the package, along with their associated SHA-256, size in bytes,
+    and the type of path (eg. hardlink vs. softlink)
+
+.. code-block:: bash
+
+  {
+    "paths": [
+      {
+        "_path": "lib/python3.7/site-packages/fun-packge/__init__.py",
+        "path_type": "hardlink",
+        "sha256": "76f3b6e34feeb651aff33ca59e0279c4eadce5a50c6ad93b961c846f7ba717e9",
+        "size_in_bytes": 2067
+      },
+      {
+        "_path": "lib/python3.7/site-packages/fun-packge/__config__.py",
+        "path_type": "hardlink",
+        "sha256": "348e3602616c1fe4c84502b1d8cf97c740d886002c78edab176759610d287f06",
+        "size_in_bytes": 87519
+      },
+      ...
+  }
+
+
+info/index.json
+---------------
+
+This file contains basic information about the package, such as
+name, version, build string, and dependencies. The content of this
+file is stored in ``repodata.json``, which is the repository
+index file, hence the name ``index.json``. The json object is a
+dictionary containing the keys shown below. The filename of the
+conda package is composed of the first 3 values, as in:
+``<name>-<version>-<build>.tar.bz2``.
+
+.. list-table::
```

Comment:
This feature is awesome! But on my screen the description section is super long and flows off the screen. Is it possible to make it flow smarter to that it all fits on the screen without needing to scroll horizontally?
I don't think this is a huge deal if it's really not straight forward to use

Reply:
I changed the width of the 3rd column so I'll see in the final product if it fixes the table overflow.